.travis.yml

```diff
@@ -34,9 +34,11 @@ matrix:
 install:
 - rvm use 2.3.1 --install --binary --fuzzy
 - bundle install --jobs=3 --retry=2 --path=vendor/bundle
-- if ! shellcheck --version ; then
-    curl -sSL "${SHELLCHECK_URL}" | tar -C "${HOME}/bin" -xjf - ;
+- if ! shellcheck --version &>/dev/null; then
+    curl -sSL "${SHELLCHECK_URL}"
+    | tar --exclude 'SHA256SUMS' --strip-components=1 -C "${HOME}/bin" -xjf -;
```

Comment:
i was unfamiliar with `--strip-components=1`  this is a more direct extraction.  TIL.  :+1: